maps/away/liberia/liberia.dmm

```diff
@@ -5019,10 +5019,8 @@
 /area/liberia/bridge)
 "jD" = (
 /obj/structure/table/glass,
-/obj/machinery/photocopier/faxmachine{
-	department = "Merchant Outpost";
+/obj/machinery/faxmachine{
 	req_access = list("ACCESS_MERCHANT");
```

Comment:
Why keep the req_access here?

Reply:
Well, access is still used for being able to use the thing. And that was what it was set to for this map.